app/views/users/_user.html.erb

```diff
@@ -1,6 +1,6 @@
 <article class="main_page_conatiner">
     <div class="image_container">
-        <%= image_tag "image_1.jpg", height: 200 %>
+        <%= image_tag "image_1.jpg", height: 200, id: user.id%>
```

Comment:
- [ ] When I try to run your app to see how the n+1 issue is fixed, I got this error as well. 👇🏾 Please look into this and fix.


<img width="946" alt="image" src="https://github.com/bilalrajput09/Blog_app_rails/assets/84156714/69f1019a-7b62-48ba-9a04-151272f50526">